src/ejected.config.ts

```diff
@@ -0,0 +1,39 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as webpack from 'webpack';
+
+import devConfigFactory from './dev.config';
+import distConfigFactory from './dist.config';
+import testConfigFactory from './test.config';
+
+export interface EnvOptions {
+	mode?: 'dev' | 'dist' | 'test';
+}
+
+const basePath = process.cwd();
+
+function loadRc() {
+	const rcPath = path.join(basePath, '.dojorc');
```

Comment:
how do you feel about ejecting the .dojorc too? as in take it out of the .dojorc and inline it in here? just means the user will fully know that the .dojorc is only for the commands then?

Reply:
that also means we can use the config helper in eject rather than having to hardcode the `cli-build` portion to pull out here

Reply:
Actually that sounds really useful, 👍

Reply:
As in 1) load the rc from `helper.configuration.get()`, 2) reset the config with `helper.configuration.set({})`, and 3) inject the `rc` into the copied `ejected.config.js` source? That or some variation of that should be doable, as soon as I figure out why `helper.configuration.get()` returns `{}` from within `eject` but not `run`.

Reply:
Whether the `build-app` rc options are copied to a new file or inlined within the copied the `ejected.config.js` file, it needs to be possible to determine both the location to which `@dojo/cli` outputs ejected files and whether ejecting was successful. Currently, neither of these is possible.

If the rc options are inlined in `ejected.config.js`, then that file needs to be modified after ejection has completed. Yet `eject` in `main.ts` is called before any files have been copied. Alternatively, a new file can be generated and added to `EjectOutput.copy`, but then it needs to be deleted after ejection completes.

Is there another viable solution I'm completely missing?